tools/loc/ui.js

```diff
@@ -610,7 +610,15 @@ async function copyFilesToLangstoreEn() {
   const previewStatuses = await Promise.all(
     copyStatuses
       .filter((status) => status.success)
-      .map((status) => simulatePreview(stripExtension(status.dstPath))),
+      .map((status) => {
+        let { dstPath } = status;
+        if (dstPath.endsWith('.xlsx')) {
+          dstPath = `${dstPath.slice(0, -5)}.json`;
+        } else {
+          dstPath = stripExtension(dstPath);
+        }
```

Comment:
For assignments, I personally prefer ternary operators. Also `.slice` is a bit less declarative than using a small regex IMO
```suggestion
        dstPath = dstPath.endsWith('.xlsx') ? dstPath.replace(/\.xlsx$/, '.json') : stripExtension(dstPath);
```

Reply:
@mokimo thanks for the suggestion. Applied yours.